src/app/components/cards/Comment.jsx

```diff
@@ -372,7 +372,16 @@ class CommentImpl extends React.Component {
         const commentClasses = ['hentry'];
         commentClasses.push('Comment');
         commentClasses.push(this.props.root ? 'root' : 'reply');
-        if (hide_body || this.state.collapsed) commentClasses.push('collapsed');
+
+        if (ignore || gray) {
+            commentClasses.push('downvoted');
+            if (!hide_body) {
+                commentClasses.push('revealed');
+            }
+        }
```

Comment:
Downvoted comments behave like a collapsed comment but they are not collapsed, as you can see there is a `[-]` link on the right to actually collapse them. 
Adding two new classes for downvoted comments to make it easier to precisely select downvoted/revealed comments.